book/src/clauses/well_known_traits.md

```diff
@@ -36,6 +36,7 @@ Some common examples of auto traits are `Send` and `Sync`.
 | str             | 📚    | 📚    | ⚬     | ⚬     | ⚬   |  ⚬  |  ⚬     |  ⚬       |    ❌        |
 | never type      | 📚   |  📚   |  ✅   |  ⚬    |  ⚬    | ⚬   |   ⚬    |  ⚬       |   ❌       |
 | trait objects   | ⚬    | ⚬    | ⚬     |  ✅    | ⚬    | ⚬   | ⚬      |  ⚬       |    ⚬        |
+| functions defs  | ✅    | ✅    | ✅     | ⚬     | ⚬    |  ⚬  | ⚬      |  ⚬       |    ❌         |
```

Comment:
I actually don't know, should `FnDef`s impl `Fn`?

Reply:
[Yes](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=c91518443d32a200da4f0a299bfe9b44).